erpnext/projects/doctype/project/project.py

```diff
@@ -1405,6 +1422,17 @@ def validate_notification(self, notification_type=None, child_doctype=None, chil
 		# Return True if no conditions catch any problem
 		return True
 
+	def set_pending_quotation_total(self, update=False, update_modified=False):
+		total = frappe.db.sql("""
+			SELECT SUM(rounded_total) 
```

Comment:
- Use `base_rounded_total` (for company currency)
- Get both `base_grand_total` and `base_rounded_total` use `base_rounded_total or base_grand_total` because erpnext makes `base_rounded_total = 0` in certain cases where rounding is disabled